### PR TITLE
Remove dq2 and more atlas cleanup

### PR DESCRIPTION
**Pull request by drmarkwslater**
The first set of big Atlas removal, clean up and DQ2 -> Rucio changes. There will be more of these but I wanted to get this in as a good start.

Note this also has the required fix to the use of `_commit` in the code.

Please feel free to look over the code but most of it is removing old stuff and all of it is Atlas specific so don't worry too much about review 😄

### CODE REVIEW

**Comment by egede** [comment]
@drmarkwslater  Should you not split into two PRs the fix of the `_commit` bug from the general clean-up ...

File: python/GangaAtlas/Lib/Rucio/__init__.py [hunk 1]
```diff
@@ -0,0 +1,275 @@
...
+    Attributes:
+        dsname (str): The pattern of the dataset to check
+
+    Returns:
+        list: the list of dataset names
+    """
+
+    # strip the trailing '/'
+    if dsname.endswith('/'):
+        dsname = dsname[:-1]
```
Comment (milliams) [inline python/GangaAtlas/Lib/Rucio/__init__.py]: Perhaps use [`rstrip('/')`](https://docs.python.org/2.7/library/stdtypes.html#str.rstrip)?

File: python/GangaAtlas/Lib/Athena/AthenaLCGRTHandler.py [hunk 2]
```diff
@@ -462,16 +448,16 @@ def master_prepare( self, app, appconfig):
...
-            firstCloud = whichCloud(job.backend.requirements.sites[0])
-            for site in job.backend.requirements.sites:
-                cloud = whichCloud(site)
-                if cloud != firstCloud:
-                    printout = 'Job submission failed ! Site specified with j.backend.requirements.sites=%s are not in the same cloud !' %(job.backend.requirements.sites)
-                    raise ApplicationConfigurationError(None,printout )
+        #if job.backend.requirements.sites:
+        #    firstCloud = whichCloud(job.backend.requirements.sites[0])
+        #    for site in job.backend.requirements.sites:
+        #        cloud = whichCloud(site)
```
Comment (alexanderrichards) [inline python/GangaAtlas/Lib/Athena/AthenaLCGRTHandler.py]: if this code is commented out does it need to be kept at all?

File: python/GangaPanda/old_test/GPI/TestPanda.py [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+        fs = getLibFileSpecFromLibDS("panda.0705111709.61282.lib._8913849")
+
+        assert fs.lfn == "panda.0705111709.61282.lib._8913849.6459655377.lib.tgz"
+        assert fs.GUID == "4EA67FCA34844526B34BE9435638AB81"
+        assert fs.dataset == "panda.0705111709.61282.lib._8913849"
+        assert fs.destinationDBlock == "panda.0705111709.61282.lib._8913849"
+        assert fs.status == "ready"
+        assert fs.md5sum == "1bbb4da4"
+        assert fs.fsize == "1881494"
+        assert fs.scope == "panda"
```
Comment (alexanderrichards) [inline python/GangaPanda/old_test/GPI/TestPanda.py]: Not a major issue but this file has no newline at the end of it.

**Comment by egede** [comment]
@drmarkwslater Can you please review the long set of conflicts.

**Comment by mesmith75** [comment]
@drmarkwslater Are you looking to merge this now?

**Comment by drmarkwslater** [comment]
Nope - in fact I'm actually gradually doing the individual smaller changes in the PRs I've put in in the last few days. I'm only keeping it up-to-date so I don't miss anything 😄 Eventually everything in this PR will have been applied and I'll close this.

**Comment by egede** [comment]
@drmarkwslater Can this PR be deleted without merging?

**Comment by egede** [comment]
Closed as no reaction for multiple months,

**Pull request closed without merging**